nomad/acl.go

```diff
@@ -0,0 +1,103 @@
+package nomad
+
+import (
+	"crypto/sha1"
```

Comment:
Is it possible to use a more secure hash function than SHA-1, such as SHA-256 or SHA-3? Golang has support for both, see https://golang.org/pkg/crypto/sha256/ and https://godoc.org/golang.org/x/crypto/sha3